parser_library/test/processing/ca_instr_test.cpp

```diff
@@ -27,341 +27,247 @@ using namespace hlasm_plugin::parser_library::semantics;
 
 TEST(var_subs, gbl_instr_only)
 {
-    std::string input("   gbla var");
```

Comment:
I am not sure about this... the language is supposed to be case-insensitive in these situations, so I think we should keep the mixed case in the tests.

Reply:
No problem